TRENZ.Lib.RazorMail.Core/Models/MailHeaderCollection.cs

```diff
@@ -37,11 +37,22 @@ public class MailHeaderCollection() : Dictionary<string, object>(StringComparer.
     /// </summary>
     public const string FromKey = "From";
 
+    /// <summary>
+    /// The key for the "Importance" header.
+    /// </summary>
+    public const string ImportanceKey = "Importance";
+
     /// <summary>
     /// The keys for the mail addresses in the collection.
     /// </summary>
     public static readonly string[] AddressKeys = [ToKey, CcKey, BccKey, ReplyToKey, FromKey];
 
+    /// <summary>
+    /// Keys for headers that the <see cref="BaseSmtpMailClient"/>
+    /// implementation must set explicitly. 
+    /// </summary>
+    public static readonly string[] SpecificHandledHeaderKeys = [..AddressKeys, ImportanceKey];
```

Comment:
What about `WellKnownHeaderKeys`?

Reply:
The meaning isn't quite the same? `SpecificHandledHeaderKeys` are keys that each implementation (MailKit, System.Net.Mail) has to handle manually, in contrast to the other keys, which are simply passed on as a dict.

Reply:
right